Source/DafnyLanguageServer.Test/Various/CounterExampleTest.cs

```diff
@@ -746,11 +725,11 @@ class Value {
       var counterExamples = (await RequestCounterExamples(documentItem.Uri)).
         OrderBy(counterexample => counterexample.Position).ToArray();
       Assert.Single(counterExamples);
-      Assert.True(counterExamples[0].Variables.ContainsKey("s:set<seq<set<_System.array<int>>>>"));
-      StringAssert.Matches(counterExamples[0].Variables["s:set<seq<set<_System.array<int>>>>"], new Regex("\\{@[0-9]+ := true\\}"));
+      Assert.Contains("exists boundVar0: seq<set<array<int>>>", counterExamples[0].Assumption);
+      Assert.Contains("boundVar0 in s", counterExamples[0].Assumption);
     }
 
-    [Theory]
+    [Theory(Skip = "This test should be re-enabled once counterexamples support arrays")]
```

Comment:
Have now added support for constraining arrays! They require some special processing since they are some array-specific AST nodes but it should all work now.